sdk/monitor/azure-monitor-ingestion/swagger/README.md

```diff
@@ -2,13 +2,19 @@
 
 ## Code generation settings
 
+### Manual Modifications
+
+The following edits need to be made manually after code generation:
+- Rollback the edits to `module-info` file
```

Comment:
Because this is a DPG client, every time you regenerate from swagger it attempts to set the module-info as well. However, it does so incorrectly and will produce build errors if it's not corrected.

Reply:
@srnagar After looking at the autorest documentation I didn't see a flag that would disable edits to the module-info.